snow/engine/avalanche/getter/getter.go

```diff
@@ -25,30 +26,38 @@ import (
 // Get requests are always served, regardless node state (bootstrapping or normal operations).
 var _ common.AllGetsServer = (*getter)(nil)
 
-func New(storage vertex.Storage, commonCfg common.Config) (common.AllGetsServer, error) {
+func New(
+	ctx *snow.ConsensusContext,
```

Comment:
Should we avoid passing in the entire context and just pass in the log + registerer?